dev/BUILD.yaml

```diff
@@ -11,14 +11,12 @@ packages:
       - dev/gpctl:app
       - dev/loadgen:app
       - dev/gp-gcloud:app
-      - dev/preview/previewctl:cli
```

Comment:
We don't add previewctl in the leeway.Dockerfile like we do with the rest so there's no need to build it here.